android/main/java/com/couchbase/lite/internal/CouchbaseLiteInternal.java

```diff
@@ -95,7 +95,7 @@ public static void init(
 
         System.loadLibrary(LITECORE_JNI_LIBRARY);
 
-        if (DEBUGGING) { C4Base.debug(); }
+        C4Base.debug();
```

Comment:
Can you remind me what C4Base.debug() is doing?

Reply:
It enables the backtrace handler for logging, and turns on "warnOnErrors"

Reply:
CBL-1566

Reply:
Thanks. The code is right below.